src/Services/OrderFormService.php

```diff
@@ -16,7 +17,10 @@
 use Railroad\Ecommerce\Gateways\PayPalPaymentGateway;
 use Railroad\Ecommerce\Repositories\PaymentMethodRepository;
 use Railroad\Ecommerce\Repositories\PaymentRepository;
+use Railroad\Ecommerce\Repositories\ProductRepository;
 use Railroad\Ecommerce\Requests\OrderFormSubmitRequest;
+use Railroad\Referral\Models\Referrer;
```

Comment:
Do we not need to include railroad/referral as a dependency in composer?

Reply:
It was working without it, but I've added and did some testing, no conflicts between other dependencies. So just to make sure everything is correct, I'll keep the dependency in. Should be good now!

Reply:
Thanks!